pkg/controller/types/group.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+import (
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/vmware-tanzu/antrea/pkg/apis/controlplane"
+)
+
+type GroupReference struct {
+	// Namespace of the Group. It's empty for ClusterGroup.
+	Namespace string
+	// Name of the Group.
+	Name string
+}
+
+// GroupSelector describes how to select GroupMembers.
+type GroupSelector struct {
+	// The normalized name is calculated from Namespace, PodSelector, ExternalEntitySelector and NamespaceSelector.
+	// If multiple policies have same selectors, they should share this group by comparing NormalizedName.
```

Comment:
that comment looks like it was copied from the `GroupSelector` type in `pkg/controller/types/networkpolicy.go`. It makes sense in the context of selectors embedded in a NetworkPolicy, but not so much in the context of ClusterGroups.

Reply:
actually this is moved from types/networkpolicy.go to types/group.go
they both share the same GroupSelector struct. Updated the text to reflect its usage for ClusterGroups